### DISCUSSION TITLE
Custom integration no longer maintained

### DISCUSSION THREAD

**Discussion by gerard33** [custom-components/media_player.braviatv_psk#75] [General]
This component is no longer maintained as I switched to the HA Sony Bravia [integration](https://www.home-assistant.io/integrations/braviatv/) which now also supports using the Pre-Shared Key.  
I can still merge PRs made and tested by the community and release new versions.
A huge thanks to all users and contributors :thumbsup:  
It was really fun to develop this custom integration and see that so many people use(d) it :tv:

**Comment by colinjgibbs** [comment]
Just wanted to say thanks for this excellent integration which I used for some years. I too have switched to the HA integration which is fine, except that it doesn't show the media info when in an app, say Netflix. Did you find a way to do this? I use the custom mini card by kalkih.

**Comment by gerard33** [comment]
Thanks for your message! I haven't tried it as I am not missing that in the HA integration.

**Comment by kroon040** [comment]
On the version of HA, it doesn't  work anymore (2025.11. The integrated version doesn't support getting the channellist  in a drop down. Shame that this version is not supported anymore
